__tests__/cache-utils.test.ts

```diff
@@ -51,7 +40,35 @@ describe('cache-utils', () => {
     });
   });
 
+  it('isCacheFeatureAvailable is false', () => {
```

Comment:
in the description, Please add whether the scenario is of GHES or not.

Reply:
Done. Thank you.